ci/run.sh

```diff
@@ -15,6 +15,9 @@ fi
 
 mkdir build && cd build
 conan install .. --build missing --profile release
+export CFLAGS='-fsanitize=address'
```

Comment:
I think it would it make more sense to set these flags in the CMake invocation. Basically you would need to add to `CMAKE_OPTIONS` the following:

```
-DCMAKE_CXX_FLAGS="-fsanitize=address" -DCMAKE_EXE_LINKER_FLAGS="-fsanitize=address" -DCMAKE_MODULE_LINKER_FLAGS="-fsanitize=address"
```
In that way, we would not be modifying any environment variable.